sites/public/pages/resources.tsx

```diff
@@ -0,0 +1,41 @@
+import { PageHeader, t } from "@bloom-housing/ui-components"
+import Layout from "../layouts/application"
+
+const Feedback = () => {
+  const pageTitle = <>{t("pageTitle.resources")}</>
+
+  return (
+    <Layout>
+      <PageHeader title={pageTitle} inverse />
+      <div className="px-5 max-w-5xl md:m-auto">
+        <p className="mt-8">{t("resources.body1")}</p>
+        <div className="mt-8">
+          <h2>{t("resources.evictionAssistance")}</h2>
+        </div>
+        <div className="mt-8">
+          <h2>{t("resources.detroitHousingNetwork")}</h2>
+          <p>
+            {t("resources.detroitHousingNetworkBody")}{" "}
+            {/* It isn't ideal to make the url separate from the translated text but polyglot
```

Comment:
Could you use interpolation here? https://github.com/airbnb/polyglot.js#interpolation

Reply:
I tried that, but it was resulting in printing out [object Object] so I think they only allow strings for interpolation and not anything else

Reply:
Wait, so substituting the string `"<a href="https://detroithousingnetwork.org">https://detroithousingnetwork.org</a>"` doesn't work? Like literally putting in the string representation of the html link?

Reply:
Putting in that as a string will result in it outputting the literal string `"<a href="https://detroithousingnetwork.org">https://detroithousingnetwork.org</a>"` but we want it to be the semantic html tag which doesn't work.

Looks like there's an open issue on polyglot to add this: https://github.com/airbnb/polyglot.js/issues/93

Sounds like we could add wrapper interpolations and manually call `dangerouslySetInnerHTML` but I'm not sure that's worth it for this. It's probably better to wait until polyglot officially supports it.

Reply:
Ah, okok, bummer. :(